x-pack/plugins/security/server/authorization/privileges/feature_privilege_builder/cases.test.ts

```diff
@@ -69,25 +76,20 @@ describe(`cases`, () => {
           },
         });
 
-        expect(casesFeaturePrivilege.getActions(privilege, feature)).toMatchInlineSnapshot(`
-          Array [
-            "cases:1.0.0-zeta1:observability/getCase",
-            "cases:1.0.0-zeta1:observability/getComment",
-            "cases:1.0.0-zeta1:observability/getTags",
-            "cases:1.0.0-zeta1:observability/getReporters",
-            "cases:1.0.0-zeta1:observability/getUserActions",
-            "cases:1.0.0-zeta1:observability/findConfigurations",
-          ]
-        `);
+        expect(casesFeaturePrivilege.getActions(privilege, feature)).toMatchSnapshot();
```

Comment:
Unfortunately, `it.each` doesn't support `toMatchLineSnapshot` so I switched it to use `toMatchSnapshot`.